daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/transaction/PartialTransaction.scala

```diff
@@ -69,27 +69,34 @@ private[lf] object PartialTransaction {
     */
   final case class Context(
       info: ContextInfo,
+      oldChild: Boolean, //one or more children predate exceptions
       children: BackStack[NodeId],
       nextActionChildIdx: Int,
   ) {
-    def addActionChild(child: NodeId): Context =
-      Context(info, children :+ child, nextActionChildIdx + 1)
+    def addActionChild(child: NodeId, version: TxVersion): Context = {
+      import scala.Ordering.Implicits.infixOrderingOps
+      val oldChild2 = oldChild || (version < TxVersion.minExceptions)
```

Comment:
I think I’m missing something. Consider the following
1. rollback node
1.1 exercise node version 1.1.dev
1.1.1 create node version 1.1.dev

The exercise node will have the flag set but I don’t see where we propagate it to the rollback node. What am I missing?

Reply:
Yes, There is a bug if we have an old-create inside a new-exercise inside a rollback.
(I think this is the example you mean)

We definitely need more testcase for these more subtle examples.
(Now we have Rem'y parser fix I will try to write them)

Reply:
code fixed now; still need tests.

Reply:
tests added